ansible/group_vars/all/vars.yml

```diff
@@ -1,2 +1,3 @@
 datadog_enabled: no
+api_gateway: no
```

Comment:
A better variable name would be "local_health_check". Also what about using vars/aeternity/* ? I hope we'll remove ansible group vars usage soon, it does not scale well for our usage.